.github/workflows/docs-check.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Check helm-docs
         run: make check-helm-docs
       - name: Check docs
-        run: make check-docs
+        run: make check.docs
```

Comment:
All other changes look good. But I'm not too sure about the target naming convention of `make check.docs`.  So I'll wait for other's opinion on this as well.

Reply:
@sp98  wrote:

> All other changes look good. But I'm not too sure about the target naming convention of `make check.docs`. So I'll wait for other's opinion on this as well.

I don't claim that `check.docs` is even  a convention. In rook, we have a few other target groups stuctured with dots, like `go.vet`, `go.fmt`, `go.mod.check, etc.

So I thought I'd introduce a new case of this pattern with  `check.foo`.

Reply:
@obnoxxx I'm not able to find doc to support my case but in rook we also, `make docs.build` `make docs.preview` etc, so not sure which is right format for this